README.md

```diff
@@ -176,6 +176,31 @@ It also is possible to check for realm roles and application roles.
 * `hasRole('realm:admin')` will check the logged in user has the admin realm role
 * `hasRole('some-other-app:admin')` will check the loged in user has the admin realm role in a different application
 
+### Error Codes
+
+Library will return specific GraphQL errors to the client that can
+be differenciated by using error codes.
+
+Example response from GraphQL Server could look as follows:
+
+```json
+{
+   "errors":[
+      {
+        "message":"User is not authorized. Must have one of the following roles: [admin]",
+        "code": "FORBIDDEN"
+      }
+   ]
+}
+```
+
+Possible error codes: 
+
+- `NOT_AUTHENTICATED`: returned when user is not authorized to access API because it requires login
+- `FORBIDDEN`: returned when user do not have permission to perform operation 
+- `ACCESS_DENIED`: returned when client did not provided any authentication information. U
```

Comment:
Is there value in having this error code?

Reply:
This one is confusing to me. Either the client is not authenticated because they provided no credentials, or there was a misconfiguration, or they are authenticated but they do not have the appropriate permissions. I think we can get rid of the `ACCESS_DENIED` code

Reply:
Ok. Doing that